### DISCUSSION TITLE
Rejected request: Incomplete profile and Distance Learner

### DISCUSSION THREAD

**Discussion by 0Architect** [community/community#57826] [GitHub Education]
### Hi everyone,

I am currently applying for the GitHub Global Campus and I keep getting rejected for the same reasons:

1. You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

2. You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

However, I have provided my name in my GitHub profile as it appears and as per my location, I'm currently doing my industrial training, as such I am not in the same state as my school. I have provided my email and an image of my identification card as a student, which contained my name, course, Matric Number, school, year I commenced my studies and the year I would end. I am not sure how to go around this process.

Your help would be much appreciated, thank you.

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
